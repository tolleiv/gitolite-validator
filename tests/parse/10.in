repo foo bar

    RW+C                     =   alice @teamleads

repo foo bahar
    # a comment
    RW+                     =   alice @teamleads
