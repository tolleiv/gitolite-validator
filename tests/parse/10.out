root:
  repo:
    paths:
      foo
      bar
    rules:
      rule:
        permission: RW+C
        members:
          alice
          @teamleads
  repo:
    paths:
      foo
      bahar
    rules:
      rule:
        permission: RW+
        members:
          alice
          @teamleads
